<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanFrame</class>
 <widget class="QFrame" name="scanStabFrame">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="scanGroupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="scanGroupBoxLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="stepSizeLabel">
         <property name="text">
          <string>Step size :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="nbrPointsLabel">
         <property name="text">
          <string>Nbr steps :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="centralPosLabel">
         <property name="text">
          <string>Central Pos :</string>
         </property>
        </widget>
       </item>

       <item row="1" column="3">
        <widget class="QComboBox" name="mvtTypeComboBox"/>
     	 <property name="text">
      	  <string>Forward</string>
         </property>
        
       </item>
       
       <item row="2" column="1">
        <widget class="QSpinBox" name="nbrPointsSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>

       <item row="2" column="3">
        <widget class="QSpinBox" name="stepDurationSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>

       <item row="0" column="1">
        <widget class="QSpinBox" name="centralPosSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-2147483647</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       
       <item row="1" column="1">
        <widget class="QSpinBox" name="stepSizeSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-6000000</number>
         </property>
         <property name="maximum">
          <number>6000000</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       
       <item row="1" column="2">
        <widget class="QLabel" name="mvtTypeLabel">
         <property name="text">
          <string>Mvt Type :</string>
         </property>
        </widget>
       </item>


       <item row="2" column="2">
        <widget class="QLabel" name="stepDurationLabel">
         <property name="text">
          <string>Nbr of acq./step :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>


     <item>
      <widget class="QLineEdit" name="dataFolderEdit">
       <property name="text">
        <string>C:\Users\mluttmann\Documents\Python Scripts\data\monscan</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Time remaining (s)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>


     <item>
      <widget class="QLCDNumber" name="timeLCD">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>26</y>
         <width>100</width>
         <height>24</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>10</number>
       </property>
      </widget>
     </item>

    </layout>
   </widget>
  </widget>





  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stab Scan</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="startScanPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Stabilized Scan</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>centralPosSpinBox</tabstop>
  <tabstop>stepSizeSpinBox</tabstop>
  <tabstop>nbrPointsSpinBox</tabstop>
  <tabstop>dataFolderEdit</tabstop>
  <tabstop>mvtTypeComboBox</tabstop>
 
  <tabstop>startScanPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
